src/routes/volunteers.js

```diff
@@ -0,0 +1,11 @@
+const router = require('express').Router();
+// const { isAuth } = require('../middlewares/auth');
+// const { request } = require('chai');
+
+const volunteersController = require('../controllers/volunteers');
+
+router.get('/', volunteersController.getVolunteers);
+
+router.get('/:id', volunteersController.getVolunteers);
```

Comment:
also you should use the auth middleware on both of these routes since the user needs to e authenticated to find a volunteers